dashboard/lib/contact_rollups_v2.rb

```diff
@@ -228,6 +232,42 @@ def upload_metrics
     Cdo::Metrics.push('ContactRollupsV2', aws_metrics)
   end
 
+  # @return [String] url of the uploaded S3 object
+  def upload_to_s3
+    log_name = "crv2-#{Time.now.utc.strftime('%Y%m%dT%H%M%SZ')}.log"
+    AWS::S3::LogUploader.
+      new('cdo-audit-logs', "contact-rollups-v2/#{CDO.rack_env}").
+      upload_log(log_name, @log_collector.to_s)
+  end
+
+  # @return [Boolean, nil] true/false if a summary message is sent to Slack or not,
+  #   nil if CDO.hip_chat_logging is not enabled.
+  def report_to_slack(log_url: nil)
+    duration = @log_collector.metrics.values_at(
+      :CollectContactsDuration,
+      :ProcessContactsDuration,
+      :SyncNewContactsDuration,
+      :SyncUpdatedContactsDuration
+    ).compact.sum
+    formatted_duration = Time.at(duration).utc.strftime("%Hh:%Mm:%Ss")
+
+    log_link = "<a href='#{log_url}'>☁ Log on S3</a>"
+
+    cloud_watch_link =
+      "<a href='https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#dashboards:name=Contact-Rollups-V2'>"\
+      "📈 CloudWatch dashboard</a>"
```

Comment:
Is it possible to use a [Slack emoji code](https://slack.com/help/articles/202931348-Use-emoji-and-emoticons) here instead of a literal emoji?